2018-09-29 12:31:05.869 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:31:05.872 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:32:05.869 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:32:05.872 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:32:48.204 15618 DEBUG oslo_concurrency.lockutils [req-a213e774-d892-4d86-b8b9-fec87e25fe58 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:32:48.597 15618 DEBUG neutron.agent.dhcp.agent [req-362f1a92-b3d7-44f1-a714-d658ef7a8e68 - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: enable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:32:48.599 15618 DEBUG neutron.agent.linux.utils [req-362f1a92-b3d7-44f1-a714-d658ef7a8e68 - - - - -] Unable to access /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:48.600 15618 DEBUG neutron.agent.linux.dhcp [req-362f1a92-b3d7-44f1-a714-d658ef7a8e68 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-20e31254-cc7e-4328-95ef-8f13064d533b on network 20e31254-cc7e-4328-95ef-8f13064d533b does not yet exist. Checking for a reserved port. _setup_reserved_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1272
2018-09-29 12:32:48.601 15618 DEBUG neutron.agent.linux.dhcp [req-362f1a92-b3d7-44f1-a714-d658ef7a8e68 - - - - -] DHCP port dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-20e31254-cc7e-4328-95ef-8f13064d533b on network 20e31254-cc7e-4328-95ef-8f13064d533b does not yet exist. Creating new one. _setup_new_dhcp_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1293
2018-09-29 12:32:51.490 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.506 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'add', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.523 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'sysctl', '-w', 'net.ipv4.conf.all.promote_secondaries=1'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.539 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', 'link', 'set', 'lo', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.552 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'sysctl', '-w', 'net.ipv6.conf.default.accept_ra=0'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.565 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', 'link', 'set', 'tap594d0ad7-67', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.587 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-o', 'link', 'show', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.601 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 12:32:51.624 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=None, port=tap594d0ad7-67) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:32:51.625 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Transaction caused no change do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:114
2018-09-29 12:32:51.627 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): AddPortCommand(bridge=br-int, may_exist=False, port=tap594d0ad7-67) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:32:51.628 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=1): DbSetCommand(table=Port, col_values=(('tag', 4095),), record=tap594d0ad7-67) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:32:51.629 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=2): DbSetCommand(table=Interface, col_values=(('type', 'internal'), ('external_ids', {'iface-id': u'594d0ad7-67e5-47f3-b775-f58f851d6e64', 'iface-status': 'active', 'attached-mac': u'fa:16:3e:d2:a7:ed'})), record=tap594d0ad7-67) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:32:51.654 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap594d0ad7-67', 'address', 'fa:16:3e:d2:a7:ed'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.664 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.673 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'link', 'set', 'tap594d0ad7-67', 'netns', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.701 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', 'link', 'set', 'tap594d0ad7-67', 'mtu', '1450'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.717 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', 'link', 'set', 'tap594d0ad7-67', 'up'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.729 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Acquired semaphore "iptables-qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:32:51.731 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'iptables-save'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.750 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'iptables-restore', '-n'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.768 15618 DEBUG neutron.agent.linux.iptables_manager [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] IPTablesManager.apply completed with success. 56 iptables commands were issued _apply_synchronized /usr/lib/python2.7/site-packages/neutron/agent/linux/iptables_manager.py:601
2018-09-29 12:32:51.769 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Releasing semaphore "iptables-qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:32:51.770 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-o', 'link', 'show', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.782 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', 'addr', 'show', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.797 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'addr', 'add', '11.37.1.2/24', 'scope', 'global', 'dev', 'tap594d0ad7-67', 'brd', '11.37.1.255'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.808 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.821 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Setting IPv4 gateway for dhcp netns on net 20e31254-cc7e-4328-95ef-8f13064d533b to 11.37.1.1 _set_default_route_ip_version /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1169
2018-09-29 12:32:51.823 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'replace', 'default', 'via', '11.37.1.1', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.833 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.844 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Cleaning stale devices for network 20e31254-cc7e-4328-95ef-8f13064d533b _cleanup_stale_devices /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1374
2018-09-29 12:32:51.845 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.855 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'find', '/sys/class/net', '-maxdepth', '1', '-type', 'l', '-printf', '%f '] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.870 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Building initial lease file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/leases _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:626
2018-09-29 12:32:51.872 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Done building initial lease file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/leases with contents:
1538325171 fa:16:3e:d2:a7:ed 11.37.1.2 * *
 _output_init_lease_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:654
2018-09-29 12:32:51.873 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:32:51.874 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:32:51.876 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Unable to access /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:51.877 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'dnsmasq', '--no-hosts', '--no-resolv', '--strict-order', '--except-interface=lo', '--pid-file=/var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/pid', '--dhcp-hostsfile=/var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host', '--addn-hosts=/var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/addn_hosts', '--dhcp-optsfile=/var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/opts', '--dhcp-leasefile=/var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/leases', '--dhcp-match=set:ipxe,175', '--bind-interfaces', '--interface=tap594d0ad7-67', '--dhcp-range=set:tag0,11.37.1.0,static,255.255.255.0,86400s', '--dhcp-option-force=option:mtu,1450', '--dhcp-lease-max=256', '--conf-file=', '--domain=openstacklocal'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:51.903 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:51.903 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:51.904 15618 DEBUG neutron.agent.linux.external_process [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:32:51.905 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:32:51.906 15618 DEBUG oslo_concurrency.lockutils [req-a213e774-d892-4d86-b8b9-fec87e25fe58 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:32:52.788 15618 DEBUG neutron.agent.dhcp.agent [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:32:52.790 15618 DEBUG neutron.agent.linux.dhcp [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:32:52.792 15618 DEBUG neutron.agent.linux.dhcp [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:32:52.797 15618 DEBUG neutron.agent.linux.utils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.810 15618 DEBUG neutron.agent.linux.dhcp [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:32:52.811 15618 DEBUG neutron.agent.linux.utils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.822 15618 DEBUG neutron.agent.linux.utils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.836 15618 DEBUG neutron.agent.linux.utils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:52.837 15618 DEBUG neutron.agent.linux.utils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:52.837 15618 DEBUG neutron.agent.linux.external_process [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:32:52.838 15618 DEBUG oslo_concurrency.lockutils [req-2d5c1ae1-9e05-497e-babc-771f7da09c7e - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:32:52.839 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:32:52.839 15618 DEBUG neutron.agent.dhcp.agent [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Discarding stale port update: admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:32:48Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-20e31254-cc7e-4328-95ef-8f13064d533b, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.2'}], id=594d0ad7-67e5-47f3-b775-f58f851d6e64, mac_address=fa:16:3e:d2:a7:ed, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:32:49Z port_update_end /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:410
2018-09-29 12:32:52.840 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:32:52.841 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:32:52.841 15618 INFO neutron.agent.dhcp.agent [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T16:32:48Z, description=, device_id=dhcpcd8f5f9f-e3e8-569f-87ef-f03c6cfc29bc-20e31254-cc7e-4328-95ef-8f13064d533b, device_owner=network:dhcp, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.2'}], id=594d0ad7-67e5-47f3-b775-f58f851d6e64, mac_address=fa:16:3e:d2:a7:ed, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:32:50Z
2018-09-29 12:32:52.843 15618 DEBUG neutron.agent.dhcp.agent [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:32:52.844 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:32:52.846 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:32:52.849 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.861 15618 DEBUG neutron.agent.linux.dhcp [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:32:52.862 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.876 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:32:52.890 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:52.891 15618 DEBUG neutron.agent.linux.utils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:32:52.891 15618 DEBUG neutron.agent.linux.external_process [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:32:52.892 15618 DEBUG oslo_concurrency.lockutils [req-524dea17-be8f-424e-9299-10aff0f60ce9 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:05.870 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:33:05.872 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:33:21.127 15618 DEBUG oslo_concurrency.lockutils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:21.129 15618 INFO neutron.agent.dhcp.agent [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:33:19Z, description=, device_id=2f15d5c4-0219-4e27-a5b8-fae34308c9d4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.1'}], id=05591445-6c34-4108-a853-aa0ea16738b6, mac_address=fa:16:3e:12:da:72, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:20Z
2018-09-29 12:33:21.130 15618 DEBUG neutron.agent.dhcp.agent [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:21.132 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:21.135 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:21.141 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:21.154 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:21.156 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:21.172 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:21.185 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:21.187 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:21.187 15618 DEBUG neutron.agent.linux.external_process [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:21.188 15618 DEBUG oslo_concurrency.lockutils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:22.040 15618 DEBUG oslo_concurrency.lockutils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:22.041 15618 INFO neutron.agent.dhcp.agent [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:33:19Z, description=, device_id=2f15d5c4-0219-4e27-a5b8-fae34308c9d4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.1'}], id=05591445-6c34-4108-a853-aa0ea16738b6, mac_address=fa:16:3e:12:da:72, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=3, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:20Z
2018-09-29 12:33:22.042 15618 DEBUG neutron.agent.dhcp.agent [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:22.043 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:22.045 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:22.049 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:22.060 15618 DEBUG neutron.agent.linux.dhcp [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:22.062 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:22.073 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:22.087 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:22.088 15618 DEBUG neutron.agent.linux.utils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:22.089 15618 DEBUG neutron.agent.linux.external_process [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:22.089 15618 DEBUG oslo_concurrency.lockutils [req-3ac424ea-c4a6-46fa-9093-17c957765e24 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:24.327 15618 DEBUG oslo_concurrency.lockutils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:24.328 15618 INFO neutron.agent.dhcp.agent [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:33:19Z, description=, device_id=2f15d5c4-0219-4e27-a5b8-fae34308c9d4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.1'}], id=05591445-6c34-4108-a853-aa0ea16738b6, mac_address=fa:16:3e:12:da:72, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=4, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:21Z
2018-09-29 12:33:24.329 15618 DEBUG neutron.agent.dhcp.agent [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:24.330 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:24.332 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:24.336 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.352 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:24.353 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.368 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.381 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:24.382 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:24.383 15618 DEBUG neutron.agent.linux.external_process [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:24.384 15618 DEBUG oslo_concurrency.lockutils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:24.900 15618 DEBUG oslo_concurrency.lockutils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:24.900 15618 INFO neutron.agent.dhcp.agent [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T16:33:19Z, description=, device_id=2f15d5c4-0219-4e27-a5b8-fae34308c9d4, device_owner=network:router_interface, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.1'}], id=05591445-6c34-4108-a853-aa0ea16738b6, mac_address=fa:16:3e:12:da:72, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=False, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=6, security_groups=[], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:24Z
2018-09-29 12:33:24.902 15618 DEBUG neutron.agent.dhcp.agent [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:24.903 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:24.905 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:24.909 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.923 15618 DEBUG neutron.agent.linux.dhcp [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:24.925 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.936 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:24.948 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:24.949 15618 DEBUG neutron.agent.linux.utils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:24.950 15618 DEBUG neutron.agent.linux.external_process [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:24.951 15618 DEBUG oslo_concurrency.lockutils [req-e22df6f2-a6df-4141-9b79-ed829c7425b4 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:32.283 15618 DEBUG oslo_concurrency.lockutils [req-6717ae95-3bb4-4152-b87a-3b787fc7a4ee e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:32.284 15618 DEBUG oslo_concurrency.lockutils [req-6717ae95-3bb4-4152-b87a-3b787fc7a4ee e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Releasing semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:35.499 15618 DEBUG oslo_concurrency.lockutils [req-1b0d217e-acf8-4468-93ef-096ea36b0642 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:35.500 15618 DEBUG oslo_concurrency.lockutils [req-1b0d217e-acf8-4468-93ef-096ea36b0642 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:36.026 15618 DEBUG oslo_concurrency.lockutils [req-1b0d217e-acf8-4468-93ef-096ea36b0642 - - - - -] Acquired semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:36.027 15618 DEBUG oslo_concurrency.lockutils [req-1b0d217e-acf8-4468-93ef-096ea36b0642 - - - - -] Releasing semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:51.970 15618 DEBUG oslo_concurrency.lockutils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:51.971 15618 INFO neutron.agent.dhcp.agent [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:33:50Z, description=, device_id=bd40e5d1-2f55-4ac6-bea0-e5137a520064, device_owner=, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.7'}], id=9af16ebe-d496-49b7-a945-9b035f8d8570, mac_address=fa:16:3e:73:1d:9a, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=True, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=3, security_groups=[u'5ef9a718-3719-4648-a2a5-bef218605e55'], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:50Z
2018-09-29 12:33:51.972 15618 DEBUG neutron.agent.dhcp.agent [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:51.975 15618 DEBUG neutron.agent.linux.dhcp [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:51.986 15618 DEBUG neutron.agent.linux.dhcp [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:51.994 15618 DEBUG neutron.agent.linux.utils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:52.011 15618 DEBUG neutron.agent.linux.dhcp [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:52.012 15618 DEBUG neutron.agent.linux.utils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:52.028 15618 DEBUG neutron.agent.linux.utils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:52.040 15618 DEBUG neutron.agent.linux.utils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:52.041 15618 DEBUG neutron.agent.linux.utils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:52.042 15618 DEBUG neutron.agent.linux.external_process [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:52.043 15618 DEBUG oslo_concurrency.lockutils [req-f88c490d-752d-4c3f-acbf-0142f135b8b7 e840e540c5714c6c96c2c110f6b763f4 5c09a2d022d64dc3aa75647b22a5295a - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:53.813 15618 DEBUG oslo_concurrency.lockutils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:53.814 15618 INFO neutron.agent.dhcp.agent [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=, binding:vif_type=unbound, binding:vnic_type=normal, created_at=2018-09-29T16:33:50Z, description=, device_id=bd40e5d1-2f55-4ac6-bea0-e5137a520064, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.7'}], id=9af16ebe-d496-49b7-a945-9b035f8d8570, mac_address=fa:16:3e:73:1d:9a, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=True, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=4, security_groups=[u'5ef9a718-3719-4648-a2a5-bef218605e55'], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:52Z
2018-09-29 12:33:53.815 15618 DEBUG neutron.agent.dhcp.agent [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:53.817 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:53.820 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:53.823 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:53.836 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:53.838 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:53.850 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:53.863 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:53.864 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:53.865 15618 DEBUG neutron.agent.linux.external_process [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:53.866 15618 DEBUG oslo_concurrency.lockutils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:33:54.388 15618 DEBUG oslo_concurrency.lockutils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:33:54.389 15618 INFO neutron.agent.dhcp.agent [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Trigger reload_allocations for port admin_state_up=True, allowed_address_pairs=[], binding:host_id=localhost.localdomain, binding:profile=, binding:vif_details=datapath_type=system, ovs_hybrid_plug=True, port_filter=True, binding:vif_type=ovs, binding:vnic_type=normal, created_at=2018-09-29T16:33:50Z, description=, device_id=bd40e5d1-2f55-4ac6-bea0-e5137a520064, device_owner=compute:nova, extra_dhcp_opts=[], fixed_ips=[{u'subnet_id': u'748725a5-2474-4860-9e5e-d7bc160fcc11', u'ip_address': u'11.37.1.7'}], id=9af16ebe-d496-49b7-a945-9b035f8d8570, mac_address=fa:16:3e:73:1d:9a, name=, network_id=20e31254-cc7e-4328-95ef-8f13064d533b, port_security_enabled=True, project_id=5c09a2d022d64dc3aa75647b22a5295a, revision_number=5, security_groups=[u'5ef9a718-3719-4648-a2a5-bef218605e55'], status=DOWN, tags=[], tenant_id=5c09a2d022d64dc3aa75647b22a5295a, updated_at=2018-09-29T16:33:54Z
2018-09-29 12:33:54.390 15618 DEBUG neutron.agent.dhcp.agent [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:33:54.391 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:33:54.394 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:33:54.399 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:54.415 15618 DEBUG neutron.agent.linux.dhcp [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:33:54.416 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:54.432 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:33:54.444 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:54.445 15618 DEBUG neutron.agent.linux.utils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:33:54.445 15618 DEBUG neutron.agent.linux.external_process [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:33:54.446 15618 DEBUG oslo_concurrency.lockutils [req-9f3d7789-d519-4181-b650-4d05130aa21b 52db2c2bb2754474a9e626ac3befb2ef 2826c499828f44719ccb8a386423ec45 - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:34:05.870 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:34:05.873 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:34:18.257 15618 DEBUG oslo_concurrency.lockutils [req-f20b3330-3375-4a2b-b125-5c31523f8f81 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:34:18.257 15618 DEBUG oslo_concurrency.lockutils [req-f20b3330-3375-4a2b-b125-5c31523f8f81 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Releasing semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:34:19.161 15618 DEBUG oslo_concurrency.lockutils [req-f20b3330-3375-4a2b-b125-5c31523f8f81 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:34:19.162 15618 DEBUG oslo_concurrency.lockutils [req-f20b3330-3375-4a2b-b125-5c31523f8f81 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Releasing semaphore "dhcp-agent-network-lock-c140d30c-ba37-4570-9e0b-48390f7eeee1" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:35:05.876 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:35:05.879 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:36:05.876 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:36:05.879 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:37:05.876 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:37:05.879 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:37:33.909 15618 DEBUG oslo_concurrency.lockutils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:37:33.910 15618 DEBUG neutron.agent.dhcp.agent [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:37:33.912 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'dhcp_release', 'tap594d0ad7-67', '11.37.1.7', 'fa:16:3e:73:1d:9a'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:37:33.931 15618 DEBUG neutron.agent.linux.dhcp [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:37:33.935 15618 DEBUG neutron.agent.linux.dhcp [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:37:33.940 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:37:33.952 15618 DEBUG neutron.agent.linux.dhcp [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:37:33.954 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:37:33.966 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:37:33.978 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:37:33.979 15618 DEBUG neutron.agent.linux.utils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:37:33.979 15618 DEBUG neutron.agent.linux.external_process [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:37:33.980 15618 DEBUG oslo_concurrency.lockutils [req-2e2e7c57-1d93-41a6-86e5-16dd5aa056dd e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:38:05.877 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:38:05.879 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 12:38:11.842 15618 DEBUG oslo_concurrency.lockutils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:38:11.843 15618 DEBUG neutron.agent.dhcp.agent [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: reload_allocations call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:38:11.844 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'dhcp_release', 'tap594d0ad7-67', '11.37.1.1', 'fa:16:3e:12:da:72'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:11.860 15618 DEBUG neutron.agent.linux.dhcp [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Building host file: /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:685
2018-09-29 12:38:11.862 15618 DEBUG neutron.agent.linux.dhcp [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Done building host file /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/host _output_hosts_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:724
2018-09-29 12:38:11.866 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['kill', '-HUP', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:11.878 15618 DEBUG neutron.agent.linux.dhcp [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Reloading allocations for network: 20e31254-cc7e-4328-95ef-8f13064d533b reload_allocations /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:514
2018-09-29 12:38:11.879 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-4', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:11.892 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['ip', 'netns', 'exec', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b', 'ip', '-6', 'route', 'list', 'dev', 'tap594d0ad7-67'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:11.905 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:11.905 15618 DEBUG neutron.agent.linux.utils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:11.906 15618 DEBUG neutron.agent.linux.external_process [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:38:11.907 15618 DEBUG oslo_concurrency.lockutils [req-2731cab8-e602-4119-9ad9-5db5e1c890ed e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:38:35.222 15618 DEBUG oslo_concurrency.lockutils [req-4c6b8eef-f39f-4707-8f11-d383ec0e9d24 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:38:35.223 15618 DEBUG neutron.agent.dhcp.agent [req-4c6b8eef-f39f-4707-8f11-d383ec0e9d24 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:38:35.226 15618 DEBUG neutron.agent.linux.utils [req-4c6b8eef-f39f-4707-8f11-d383ec0e9d24 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command (rootwrap daemon): ['kill', '-9', '23644'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:35.247 15618 DEBUG neutron.agent.linux.utils [req-4c6b8eef-f39f-4707-8f11-d383ec0e9d24 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Running command: ['ip', '-o', 'link', 'show', 'br-int'] create_process /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:92
2018-09-29 12:38:35.274 15618 DEBUG ovsdbapp.backend.ovs_idl.transaction [-] Running txn command(idx=0): DelPortCommand(if_exists=True, bridge=br-int, port=tap594d0ad7-67) do_commit /usr/lib/python2.7/site-packages/ovsdbapp/backend/ovs_idl/transaction.py:84
2018-09-29 12:38:35.359 15618 DEBUG neutron.agent.linux.utils [req-aa89fcdd-7186-4456-833a-b8eb58923006 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:35.371 15618 DEBUG neutron.agent.linux.utils [req-aa89fcdd-7186-4456-833a-b8eb58923006 - - - - -] Running command (rootwrap daemon): ['ip', 'netns', 'delete', 'qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:35.386 15618 DEBUG oslo_concurrency.lockutils [req-aa89fcdd-7186-4456-833a-b8eb58923006 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:38:43.779 15618 DEBUG oslo_concurrency.lockutils [req-aa005b55-9af3-4b50-aa70-f18b180ca4ea e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:38:44.155 15618 DEBUG neutron.agent.linux.utils [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:44.157 15618 DEBUG neutron.agent.linux.utils [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Unable to access /var/lib/neutron/external/pids/20e31254-cc7e-4328-95ef-8f13064d533b.pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:44.158 15618 DEBUG neutron.agent.linux.external_process [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:38:44.160 15618 DEBUG neutron.agent.dhcp.agent [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Calling driver for network: 20e31254-cc7e-4328-95ef-8f13064d533b action: disable call_driver /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:133
2018-09-29 12:38:44.162 15618 DEBUG neutron.agent.linux.utils [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Unable to access /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:44.163 15618 DEBUG neutron.agent.linux.utils [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Unable to access /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/pid get_value_from_file /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:254
2018-09-29 12:38:44.164 15618 DEBUG neutron.agent.linux.external_process [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] No process started for 20e31254-cc7e-4328-95ef-8f13064d533b disable /usr/lib/python2.7/site-packages/neutron/agent/linux/external_process.py:126
2018-09-29 12:38:44.164 15618 DEBUG neutron.agent.linux.dhcp [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] Unable to access /var/lib/neutron/dhcp/20e31254-cc7e-4328-95ef-8f13064d533b/interface _get_value_from_conf_file /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:271
2018-09-29 12:38:44.165 15618 DEBUG neutron.agent.linux.dhcp [req-4ad04860-7344-41c0-9c10-fb339402fdb5 - - - - -] No interface exists for network 20e31254-cc7e-4328-95ef-8f13064d533b destroy /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:1487
2018-09-29 12:38:44.201 15618 DEBUG neutron.agent.linux.utils [req-c3d03b9f-5440-4058-8292-8a9f266fb010 - - - - -] Running command (rootwrap daemon): ['ip', '-o', 'netns', 'list'] execute_rootwrap_daemon /usr/lib/python2.7/site-packages/neutron/agent/linux/utils.py:108
2018-09-29 12:38:44.215 15618 DEBUG neutron.agent.linux.dhcp [req-c3d03b9f-5440-4058-8292-8a9f266fb010 - - - - -] Namespace already deleted: qdhcp-20e31254-cc7e-4328-95ef-8f13064d533b _destroy_namespace_and_port /usr/lib/python2.7/site-packages/neutron/agent/linux/dhcp.py:249
2018-09-29 12:38:44.217 15618 DEBUG oslo_concurrency.lockutils [req-c3d03b9f-5440-4058-8292-8a9f266fb010 - - - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:38:44.602 15618 DEBUG neutron.agent.dhcp.agent [req-c90df13e-20b4-48ff-84b7-05880203bbc8 - - - - -] resync (20e31254-cc7e-4328-95ef-8f13064d533b): ['Agent port was deleted'] _periodic_resync_helper /usr/lib/python2.7/site-packages/neutron/agent/dhcp/agent.py:256
2018-09-29 12:38:44.604 15618 INFO neutron.agent.dhcp.agent [req-c90df13e-20b4-48ff-84b7-05880203bbc8 - - - - -] Synchronizing state
2018-09-29 12:38:45.193 15618 INFO neutron.agent.dhcp.agent [req-50f96eb0-cb3c-4003-a55b-b7f785104038 - - - - -] All active networks have been fetched through RPC.
2018-09-29 12:38:45.195 15618 INFO neutron.agent.dhcp.agent [-] Starting network 20e31254-cc7e-4328-95ef-8f13064d533b dhcp configuration
2018-09-29 12:38:45.195 15618 INFO neutron.agent.dhcp.agent [-] Finished network 20e31254-cc7e-4328-95ef-8f13064d533b dhcp configuration
2018-09-29 12:38:45.196 15618 INFO neutron.agent.dhcp.agent [-] Starting network c140d30c-ba37-4570-9e0b-48390f7eeee1 dhcp configuration
2018-09-29 12:38:45.197 15618 INFO neutron.agent.dhcp.agent [-] Finished network c140d30c-ba37-4570-9e0b-48390f7eeee1 dhcp configuration
2018-09-29 12:38:45.201 15618 INFO neutron.agent.dhcp.agent [req-50f96eb0-cb3c-4003-a55b-b7f785104038 - - - - -] Synchronizing state complete
2018-09-29 12:38:45.203 15618 DEBUG oslo_concurrency.lockutils [req-aa005b55-9af3-4b50-aa70-f18b180ca4ea e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Acquired semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:212
2018-09-29 12:38:45.204 15618 DEBUG oslo_concurrency.lockutils [req-aa005b55-9af3-4b50-aa70-f18b180ca4ea e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - - -] Releasing semaphore "dhcp-agent-network-lock-20e31254-cc7e-4328-95ef-8f13064d533b" lock /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:225
2018-09-29 12:39:05.878 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" acquired by "neutron.agent.linux.external_process._check_child_processes" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 12:39:05.883 15618 DEBUG oslo_concurrency.lockutils [-] Lock "_check_child_processes" released by "neutron.agent.linux.external_process._check_child_processes" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
